projects/plugins/jetpack/extensions/store/wordpress-com/types.ts

```diff
@@ -31,6 +31,7 @@ export type AiFeatureProps = {
 		nextStart: string;
 		requestsCount: number;
 	};
+	nextTier: null;
```

Comment:
I think we can define the type, too:

```suggestion
		nextTier: {
		slug: string;
		value: TierValueProp;
		limit: number;
		redeableLimit: string;
	};
```